internal/model/download.go

```diff
@@ -0,0 +1,13 @@
+package model
+
+type Download struct {
+	ReportType         string `json:"reportType"`
+	ReportJournalType  string `json:"reportJournalType"`
+	ReportScheduleType string `json:"reportScheduleType"`
+	ReportAccountType  string `json:"reportAccountType"`
+	ReportDebtType     string `json:"reportDebtType"`
+	DateField          *Date  `json:"dateOfTransaction,omitempty"`
```

Comment:
maybe rename to match the json key, this is the only different one

Reply:
done